tests/python/unittest/test_codegen_c_host.py

```diff
@@ -0,0 +1,83 @@
+import tvm
+import numpy as np
+from tvm._ffi.libinfo import find_include_path
+
+def test_add():
+    nn = 1024
+    n = tvm.convert(nn)
+    A = tvm.placeholder((n,), name='A')
+    B = tvm.placeholder((n,), name='B')
+    C = tvm.compute(A.shape, lambda *i: A(*i) + B(*i), name='C')
+    s = tvm.create_schedule(C.op)
+
+    def check_c():
+        f1 = tvm.lower(s, [A, B, C], name="fadd")
+        fsplits = [x for x in tvm.ir_pass.SplitHostDevice(f1)]
+        fsplits[0] = tvm.ir_pass.LowerTVMBuiltin(fsplits[0])
+        mhost = tvm.codegen.build_module(fsplits[0], "c")
+        mhost.export_library("mod.so")
```

Comment:
One last minor thing: you can use tvm.contrib.util.tempdir to save and load mod.so
See: https://github.com/dmlc/tvm/blob/master/tests/python/unittest/test_module_load.py#L48-L52